lib/reservation_monitor.py

```diff
@@ -98,11 +98,14 @@ def _check_flight_fares(self) -> None:
             # and continue
             try:
                 fare_checker.check_flight_price(flight)
+                self.notification_handler.healthchecks_success("Successful fare check, confirmation number=" + flight.confirmation_number)
             except RequestError as err:
+                self.notification_handler.healthchecks_fail("Failed fare check, confirmation number=" + flight.confirmation_number)
                 logger.error("Requesting error during fare check. %s. Skipping...", err)
             except FlightChangeError as err:
                 logger.debug("%s. Skipping fare check", err)
```

Comment:
@StevenMassaro when a FlightChangeError occurs (a flight cannot be changed online--a.k.a the flight's change button is greyed out), do you think a `healthchecks_success` still suffices? It would still technically succeed, but just not be able to check for flight changes and would be annoying if you get notified on Healthchecks when this error occurs (or the lack of a success ping) when the fare check works perfectly fine.

Reply:
I think you should get a success ping during a FlightChangeError. I say this because I had a flight recently that had one of these changes, and I didn't want to change my flight time, so it was getting a FlightChangeError every day for months.